vendor/VisualAcceptance.js

```diff
@@ -128,54 +153,56 @@ function getOptions (options) {
  */
 function _capture (imageName, options) {
   var browser = window.ui
-  var istargetbrowser = JSON.parse(httpGet('/istargetbrowser?' + $.param(browser)))
-  if (istargetbrowser === false) {
-    return new Promise(function (resolve) {
-      resolve('Does not match target browser')
-    })
-  }
+  return httpGet('/istargetbrowser?' + $.param(browser)).then(function (response) {
```

Comment:
Is it possible that `istargetbrowser` changes during single test session?
Looks like we can improve here if we have this value beforehand to avoid extra requests per each screenshot.